posts/coalesce2022-a-recap.md

```diff
@@ -0,0 +1,238 @@
+---
+authors:
+- Pooja Krishnan
+tags:
+- Data
+- dbt
+- Python
+- Snowflake
+date: 2022-10-31T13:33:00.000Z
+title: "Coalesce 2022 - The Analytics Engineering Conference hosted by dbt Labs (A Recap)"
+image: 
+---
+
+Coalesce 2022 is dbt Labs' analytics engineering conference. For its third year, from October 17th to 21st, 2022, Coalesce was a whirlwind data extravaganza boasting speaker sessions and workshops in In-Person in London, Sydney, and New Orleans and virtually. Over 2500 people from all over the world attended in person, and over 20,000 individuals online. I had the pleasure of attending in-person in New Orleans over the week, and throughout this blog, I'll be presenting the Top Announcements of the week from dbt and some of my favorite sessions I attended while there.
```

Comment:
The "boasting speaker sessions and workshops in In-Person in London" sounds a bit strange... is the name "In-Person" in which case this grammatically makes sense but still _feels_ weird.